.github/workflows/build-and-test.yml

```diff
@@ -11,4 +13,12 @@ jobs:
     - uses: cachix/install-nix-action@v19
       with:
         nix_path: nixpkgs=channel:nixos-unstable
+        extra_nix_config: |
+          extra-access-tokens = github.com=${{ secrets.GITHUB_TOKEN }}
```

Comment:
Is this necessary?

Reply:
See https://github.com/cachix/install-nix-action#usage-with-flakes. Says this helps with rate limiting.